edb/schema/inheriting.py

```diff
@@ -29,40 +30,70 @@
 from . import name as sn
 from . import objects as so
 from . import utils
+from edb.schema import schema as s_schema
```

Comment:
`from . import schema as s_schema` for visual consistency with other imports.  Also, please insert imports in lexicographical order.

Reply:
You are right, Elvis. I indeed used Black and made the mistake to not separate the diff chunks. I will fix as you ask, and the next time I will pay attention to not repeat.